Source/Meadow.Contracts/Hardware/Contracts/IOControllers/II2cController.cs

```diff
@@ -23,6 +23,7 @@ public interface II2cController
         /// Creates an I2C bus instance for the default pins and the requested bus speed
         /// </summary>
         /// <param name="frequency">The bus speed in (in Hz).</param>
+        /// <param name="busNumber">The bus number</param>
         /// <returns>An instance of an I2cBus</returns>
         II2cBus CreateI2cBus(
             int busNumber,
```

Comment:
should this have a default value of `0`?

Reply:
You can't because it's followed by a `Frequency` and you can't set a default value for that parameter because, well, C#.